wormhole-connect/src/AppRouter.tsx

```diff
@@ -71,6 +73,8 @@ function AppRouter({ config }: Props) {
     if (prevRoute === redeemRoute && route !== redeemRoute) {
       dispatch(clearRedeem());
       dispatch(clearWallets());
+      dispatch(clearNtt());
```

Comment:
I'm ok with this for now, but stuff like this is going to make it harder to do the SDKv2 migration. in theory `AppRouter` should never have route-specific logic in it.